test/Microsoft.AspNetCore.JsonPatch.Test/ErrorMessageFormats.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+
+namespace Microsoft.AspNetCore.JsonPatch
+{
+    public static class ErrorMessageFormats
```

Comment:
We don't generally do things like this, our general practice is to just put the text in the test.  The **reason** for this is so that you have to double check that the message makes sense in all the cases it is used when making changes. Doing things this way defeats that.

This is one of @Eilon 's rules